deployment/default.nix

```diff
@@ -256,7 +226,7 @@ let
   };
 
   envs = {
-    david = mkEnv "david" "eu-west-1" [ keys.david ];
+    david = mkEnv "david" "sa-east-1" [ keys.david ];
```

Comment:
`sa-east-1` ?

Reply:
São Paulo, it turns out trying to send gigabytes of nix closures over the atlantic ocean is not very fast.